docs/installing/adding_devices.md

```diff
@@ -24,35 +24,36 @@ The following instructions guide you through the process of installing the requi
 
 {{site.data.keyword.ieam}} supports architectures and operating systems with the following hardware architectures:
 
-* x86_64
-   * {{site.data.keyword.linux_bit_notm}} devices or virtual machines that run Ubuntu 22.x (jammy), Ubuntu 20.x (focal), Ubuntu 18.x (bionic), Debian 10 (buster), Debian 9 (stretch)
-   * {{site.data.keyword.rhel}} 8.1, 8.2, and 8.3
-   * Fedora Workstation 32
-   * CentOS 8.1, 8.2, and 8.3
-   * SuSE 15 SP2
-* ppc64le
-   * {{site.data.keyword.linux_ppc64le_notm}} devices or virtual machines that run Ubuntu 20.x (focal) or Ubuntu 18.x (bionic)
-   * {{site.data.keyword.rhel}} 7.6, 7.9, 8.1, 8.2, and 8.3
-* ARM (32-bit)
-   * {{site.data.keyword.linux_notm}} on ARM (32-bit), for example Raspberry Pi, running Raspberry Pi OS buster or bullseye
-* ARM (64-bit)
-   * {{site.data.keyword.linux_notm}} on ARM (64-bit), for example NVIDIA Jetson Nano, TX1, or TX2, running Ubuntu 18.x (bionic)
-* Mac
-   * {{site.data.keyword.macOS_notm}}
-
-**Notes**: 
-
-* Installation of edge devices with Fedora or SuSE is only supported by the [Advanced manual agent installation and registration](../installing/advanced_man_install.md) method.
-* CentOS and {{site.data.keyword.rhel}} on {{site.data.keyword.ieam}} {{site.data.keyword.version}} only support Docker as a container engine.
-* While {{site.data.keyword.ieam}} {{site.data.keyword.version}} supports running {{site.data.keyword.rhel}} 8.x with Docker, it is officially unsupported by {{site.data.keyword.rhel}}.
+- x86_64
+  - {{site.data.keyword.linux_bit_notm}} devices or virtual machines that run Ubuntu 22.x (jammy), Ubuntu 20.x (focal), Ubuntu 18.x (bionic), Debian 10 (buster), Debian 9 (stretch)
+  - {{site.data.keyword.rhel}} 8.1, 8.2, 8.3, 8.4, 8.5 and 8.6
+  - Fedora Workstation 32
+  - CentOS 8.1, 8.2, 8.3, 8.4 and 8.5
+  - SuSE 15 SP2
+- ppc64le
+  - {{site.data.keyword.linux_ppc64le_notm}} devices or virtual machines that run Ubuntu 20.x (focal) or Ubuntu 18.x (bionic)
+  - {{site.data.keyword.rhel}} 7.6, 7.9, 8.1, 8.2, and 8.3
+- ARM (32-bit)
+  - {{site.data.keyword.linux_notm}} on ARM (32-bit), for example Raspberry Pi, running Raspberry Pi OS buster or bullseye
+- ARM (64-bit)
+  - {{site.data.keyword.linux_notm}} on ARM (64-bit), for example NVIDIA Jetson Nano, TX1, or TX2, running Ubuntu 18.x (bionic)
+- Mac
+  - {{site.data.keyword.macOS_notm}}
```

Comment:
Do we need to specify Intel-only Macs, not arm-based Macs?

Reply:
I took a separate todo yesterday to open a ticket about doc changes to clarify that Open Horizon supports Intel-only Macs, not arm-based Macs.  This is a RHEL ticket so I won't fix that here.  Defer to #262